test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -146,4 +148,11 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
     public static void assertFileDoesNotExist(String filePath) {
         assertTrue(Files.notExists(Paths.get(filePath)));
     }
+
+    @Test
+    public void isAnyNullTest() throws Exception{
```

Comment:
Should have included more cases.
For 4kloc project, unit test has to be sufficiently covered. This includes malformed inputs, assert failures and assert successes.